docs/docs/snippets/attribute-kind-params.mdx

```diff
@@ -0,0 +1,34 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<!-- vale Infrahub.spelling = NO -->
+
+<Tabs>
+<TabItem value="Number">
+| Parameter | Default | Required |
```

Comment:
Should the required column really be here? Will have a parameter that will ever be required? To me it feels like they will always be optional.